task/src/main/java/com/flipkart/flux/impl/task/TaskExecutor.java

```diff
@@ -78,7 +78,12 @@ protected Event run() throws Exception {
 		}
 		final Object returnObject = result.getKey();
 		if (returnObject != null) {
-			return new Event(outputeEventName, returnObject.getClass().getCanonicalName(), Event.EventStatus.triggered, stateMachineId, objectMapper.writeValueAsString(returnObject), MANAGED_RUNTIME);
+            Event event = (Event) returnObject;
+            event.setName(outputeEventName);
```

Comment:
1. To construct the full event in LocalJVMTask I need have the parameters outputeEventName, stateMachineId in LocalJVMTask but I don't want to pass them as part of parameters.
2. To construct in Task executor I don't have Deployment Unit object Mapper.
So went with this approach, if you see any other better approach I'm fine changing it.